Model/Source/SourceItem.php

```diff
@@ -13,31 +14,56 @@ class SourceItem
      */
     private $searchCriteriaBuilder;
 
+    /**
+     * @var Manager
+     */
+    private $moduleManager;
+
     /**
      * @var SourceItemRepositoryInterface
      */
-    private $sourceItemRepository;
+    private $sourceItemRepository = null;
```

Comment:
You don't have to define null explicitly
```suggestion
    private $sourceItemRepository;
```

Reply:
I followed how the SourceItem was declared in `Model\Sales\MagentoOrderCollection` for consistency:
```
class MagentoOrderCollection extends MagentoCollection
{
    /**
     * @var \Magento\Sales\Model\ResourceModel\Order\Collection
     */
    private $orders = null;

    /**
     * @var SourceItem
     */
    private $sourceItem = null;
```
I can remove it if so desired